File: Tmp/Four/feature.patch.f90
diff --git a/eg.f90 b/eg.f90
index 79ba24f..7c08f4d 100644
--- a/eg.f90
+++ b/eg.f90
@@ -19,6 +19,9 @@ FUNCTION convert(name)
   ELSE IF (first_char == 'c') THEN
     PRINT*, 'C'
     convert = 3
+  ELSE IF (first_char == 'd') THEN
+    PRINT*, 'D'
+    convert = 4
   ENDIF
 
 

File: Tmp/Four/feature.patch.c
diff --git a/eg.c b/eg.c
index eac0d4e..a15e3d1 100644
--- a/eg.c
+++ b/eg.c
@@ -34,6 +34,9 @@ int convert(char* name){
   }else if(first_char == 'c'){
     printf("%c", 'C');
     converted_val = 3;
+  }else if(first_char == 'd'){
+    printf("%c", 'D');
+    converted_val = 4;
   }
   printf("%c", '\n');
 
